pkgs/development/interpreters/eff/default.nix

```diff
@@ -1,20 +1,19 @@
-{ stdenv, fetchgit, ocaml, findlib, ocamlbuild, menhir, which }:
+{ stdenv, fetchzip, which, ocamlPackages }:
 
-let inherit (stdenv.lib) getVersion versionAtLeast; in
-
-assert versionAtLeast (getVersion ocaml) "3.12";
+let version = "5.0"; in
 
 stdenv.mkDerivation {
 
-  name = "eff-20140928";
+  name = "eff-${version}";
 
-  src = fetchgit {
-    url = "https://github.com/matijapretnar/eff.git";
-    rev = "90f884a790fddddb51d4d1d3b7c2edf1e8aabb64";
-    sha256 = "0cqqrpvfw0nrk5d28mkzfvc8yzqxcss0k46bkmqhqjkqq886n2mm";
+  src = fetchzip {
```

Comment:
`fetchFromGithub` maybe?

Reply:
Why?

Reply:
It has the same semantics, but is it is mainly about consistency with the code base. We also have some tooling that looks for `fetchFromGithub` and interfere additional information.